content/docs/extensions/extensions-intro.md

```diff
@@ -0,0 +1,264 @@
+---
+title: Postgres extensions
+enableTableOfContents: true
+---
+
+Explore supported Postgres extensions by category. Also see:
+
+- [List view of supported extensions and versions](/docs/extensions/pg-extensions)
+- [Install an extension](/docs/extensions/pg-extensions#install-an-extension)
+- [Update an extension](/docs/extensions/pg-extensions#update-an-extension-version)
+- [Request extension support](/docs/extensions/pg-extensions#request-extension-support)
+
+## AI / Machine Learning
+
+<DetailIconCards>
+
+<a href="/docs/extensions/pg_tiktoken" description="Tokenize data in Postgres using the OpenAI tiktoken library" icon="app-store" icon="app-store">pg_tiktoken</a>
+
+<a href="/docs/extensions/pgvector" description="Store vector embeddings and perform vector similarity search in Postgres" icon="app-store" icon="app-store">pgvector</a>
+
+</DetailIconCards>
+
+## Analytics
+
+<DetailIconCards>
+
+<a href="https://github.com/citusdata/postgresql-hll" description="Implements a HyperLogLog data structure as a native data type for efficient and tunable distinct value counting" icon="app-store" icon="app-store">hll</a>
+
+<a href="/docs/extensions/timescaledb" description="Enables Postgres as a time-series database for efficient storage and retrieval of time-series data" icon="app-store">timescaledb</a>
+
+</DetailIconCards>
+
+## Auditing / Logging
+
+<DetailIconCards>
+
+<a href="https://www.postgresql.org/docs/current/contrib-spi.html" description="Implements a trigger that stores the current user's name into a text field, useful for tracking who modified a particular row within a table" icon="app-store">insert_username</a>
+
+<a href="https://www.postgresql.org/docs/current/contrib-spi.html" description="Implements a trigger that automatically updates a timestamp column to the current timestamp whenever a row is modified" icon="app-store">moddatetime</a>
+
+<a href="https://www.postgresql.org/docs/16/pgrowlocks.html" description="Provides a function that shows row locking information for a specified table, useful in concurrency and deadlock debugging" icon="app-store">pgrowlocks</a>
+
+<a href="https://www.postgresql.org/docs/16/tcn.html" description="Provides a trigger function to notify listeners of changes to a table, allowing applications to respond to changes in the database" icon="app-store">tcn</a>
+
+</DetailIconCards>
+
+## Data / Transformations
+
+<DetailIconCards>
+
+<a href="https://postgis.net/docs/Extras.html#Address_Standardizer" description="A single-line address parser that takes an input address and normalizes it based on a set of rules" icon="app-store">address_standardizer</a>
+
+<a href="https://postgis.net/docs/Extras.html#Address_Standardizer" description="Provides data for standardizing US addresses, for use with the address_standardizer extension" icon="app-store">address_standardizer_data_us</a>
```

Comment:
the link should lead to `address_standardizer_us`, shouldn't it?

Reply:
Fixed.